modules/manage/pages/iceberg/choose-iceberg-mode.adoc

```diff
@@ -0,0 +1,231 @@
+endif::[]
+
+ifdef::env-cloud[]
+In `value_schema_latest` mode, you only need to set the property value to the string `value_schema_latest`. This enables the default behavior of `value_schema_latest` mode, which determines the subject for the topic using the TopicNameStrategy. For example, if your topic is named `sensor` the schema is looked up in the `sensor-value` subject. For Protobuf data, the default behavior also deserializes records using the first message defined in the corresponding Protobuf schema stored in the Schema Registry.
+endif::[]
+
+If you use a different strategy other than the topic name to derive the subject name, you can override the default behavior of `value_schema_latest` mode and explicitly set the subject name.
+
+To override the default behavior, use the following optional syntax:
+
+[,bash]
+----
+value_schema_latest:subject=<subject-name>,protobuf_name=<protobuf-message-full-name>
+----
+
+* For both Avro and Protobuf, specify a different subject name by using the key-value pair `subject=<subject-name>`, for example `value_schema_latest:subject=sensor-data`.
+* For Protobuf only:
+** Specify a different message definition by using a key-value pair `protobuf_name=<message-name>`, for example: `value_schema_latest:protobuf_name=com.example.manufacturing.SensorData`.
+** To specify both a different subject and message definition, separate the key-value pairs with a comma, for example: `value_schema_latest:subject=my_protobuf_schema,protobuf_name=com.example.manufacturing.SensorData`.
+
+== How Iceberg modes translate to table format
+
+Redpanda generates an Iceberg table with the same name as the topic. In each mode, Redpanda writes to a `redpanda` table column that stores a single Iceberg https://iceberg.apache.org/spec/#nested-types[struct^] per record, containing nested columns of the metadata from each record, including the record key, headers, timestamp, the partition it belongs to, and its offset. 
+
+For example, if you produce to a topic `ClickEvent` according to the following Avro schema:
+
+[,avro]
+----
+{
+    "type": "record",
+    "name": "ClickEvent",
+    "fields": [
+        {
+            "name": "user_id",
+            "type": "int"
+        },
+        {
+            "name": "event_type",
+            "type": "string"
+        },
+        {
+            "name": "ts",
+            "type": "string"
+        }
+    ]
+}
+----
+
+The `key_value` mode writes to the following table format:
+
+[,sql]
+----
+CREATE TABLE ClickEvent (
+    redpanda struct<
+        partition: integer NOT NULL,
+        timestamp: timestamp NOT NULL,
+        offset:    long NOT NULL,
+        headers:   array<struct<key: binary NOT NULL, value: binary>>,
+        key:       binary
+    >,
+    value binary
+)
+----
+
+Use `key_value` mode if the topic data is in JSON or if you can use the Iceberg data in its semi-structured format.
```

Comment:
I had to read this sentence 3-4 times, and am not 100% clear on its meaning.
Use `key_value` mode if the topic data is in JSON, or if you can, use the Iceberg data in its semi-structured format.
Use `key_value` mode if the topic data is in JSON, or the Iceberg data in its semi-structured format.

Reply:
Rephrased